roles/nginx-ufw/tasks/main.yml

```
@@ -0,0 +1,8 @@
+---
+- name: allow https traffic through the firewall
+  ufw:
+    app: WWW Secure
+    rule: allow
+    comment: nginx web server
```

So long as Nginx is installed before this is ran (which it currently is), this suggestion would work and be more explicit.
```suggestion
    app: Nginx HTTPS
    rule: allow
```